server/app.js

```diff
@@ -0,0 +1,29 @@
+const express = require('express')
+const cors = require('cors');
+
+const app = express()
+const artistRoute = require('./controllers/artists');
+const helpers = require('./helpers/artistsFuncs');
+
+app.use(cors());
+app.use('/artists', artistRoute);
+
+// Route for retrieve home page message
+app.get('/', (req, res) => res.send('Hello World!'))
+
+// Route for retrieving a list of artists that match the query string
+app.get('/query/:query', (req, res) => {
+    const results = helpers.searchArtist(req.params.query);
+    try {
+        if (results != null) {
+            res.status(200).send(results);
+        } else {
+            res.status(404).send({ message: `Couldn't GET artists with search query: ${req.params.query}. No artists' names match the given string.`}); 
+        }
+    } catch(e) {
+        console.log(e);
+        res.status(404).send({ message: `Couldn't GET artists with search query: ${req.params.query}`});
+    }
+});
+
```

Comment:
When you type an unknown name, it just returns blank page - i think it runs in to some issues with other functions before implementing this?